go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/miguelmatosduarte/instrumentedsql
+
+go 1.11
```

Comment:
Can we bump this up or there is a reason for this? (just curious)

Reply:
yeah we can bump it, no problem :) 1.17 alright?

Reply:
deal